admin/replication/ProcessReplicationChanges

```diff
@@ -74,7 +74,7 @@ $sql->auto_commit;
 $sql->do('SET search_path = musicbrainz, public');
 
 $sql->auto_commit;
-$sql->do('SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE');
+$sql->do('SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL REPEATABLE READ');
```

Comment:
Yeah, "phantom read" is defined in the PostgreSQL documentation about transaction isolation levels: https://www.postgresql.org/docs/12/transaction-iso.html (I will add a link to the commit message).

Reply:
OK, IIUC this commit is dropping a constraint not because it should not be met, but because it has always been met by design and making Postgres to double-check it is taking too much memory?

Reply:
Yes, that's correct.

Reply:
I added that link to the commit message, though I also changed "phantom reads" to "serialization anomalies," because I realized that I misread the table in the link: "The table also shows that PostgreSQL's Repeatable Read implementation does not allow phantom reads."